src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -551,7 +561,16 @@ def _handle_comm_message(self, msg):
             elif r_type == 'job_logs_latest':
                 if job_id is not None:
                     num_lines = msg['content']['data'].get('num_lines', None)
-                    self._get_latest_job_logs(job_id, num_lines=num_lines)
+                    try:
+                        self._get_latest_job_logs(job_id, num_lines=num_lines)
+                    except Exception as e:
+                        print str(e)
```

Comment:
This will print out to any output area if it happens. Should use a log message if you want it captured.